### ISSUE TITLE
@math should be checked for appropriate translation in slides

### ISSUE THREAD

**Issue by ds26gte** [bootstrapworld/curriculum#1648]
E.g., `@math{R^2}` becomes `R^2^` for asciidoc output, but this is inappropriate for slides, which use markdown. Find and correct all such cases. Reported by @retabak

**Comment by schanzer** [comment]
@ds26gte is this needed, since the infrastructure will auto-generate slides from asciidoc to begin with?

**Comment by ds26gte** [comment]
Yes, whether we're writing the slides by hand in markdown with directives like @math, or we're generating the markdown from adoc, still carrying over directives like @math, we need to ensure that the @math arguments get transliterated appropriately.

**Comment by schanzer** [comment]
@retabak does Dorai's fix address the issue you reported?
